app/controllers/image_uploads_controller.rb

```diff
@@ -5,9 +5,12 @@ class ImageUploadsController < ApplicationController
   def create
     authorize :image_upload
 
+    raise "too many uploads" if RateLimitChecker.new(current_user).limit_by_situation("image_upload")
```

Comment:
This error shows up in the logs but isn't passed over to the client.

Reply:
I suggest creating a custom exception for this case.